docs/sources/structure/topic-types/visualization/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Visualization topic
+menuTitle: Visualization
+description: Learn how to write a visualization topic.
+weight: 400
+aliases:
+  - /docs/writers-toolkit/structure/topic-types/visualization
+keywords:
+  - topic types
+  - template
+  - visualization
+---
+
+# Visualization topic
+
+A _visualization_ topic provides an overview of a visualization type, its use cases, and the display options available in the user interface (UI). Visualization topics always include conceptual and reference content and may include task content.
+
+The following types of content can be included in visualization topics:
+
+- A high-level overview of a visualization's features, with use cases
+- Descriptions of the UI options unique to that visualization (common options are documented separately)
+- Screenshots that help users understand UI interactions or hard-to-describe UI
+- As needed, guidance on how to use the visualization in special use cases
+- Best practice guidelines
+
+A visualization topic does not include:
+
+- Tutorial content
+
+Most visualizations require a data source; the ones that don't are called _widgets_. As a result, a widget requires less documentation than a typical visualization. The structures for both of these are explained in the following sections:
+
+- [Visualization topic structure](#visualization-topic-structure)
+- [Widget topic structure](#widget-topic-structure)
+
+## Visualization topic structure
+
+The following applies to visualizations that require a data source. For visualizations that don't require one, refer to [Widget topic structure](#widget-topic-structure).
+
+A visualization topic includes the following elements:
+
+- **Topic title:** A visualization topic title is the same as the name of the visualization as it's written in the UI (for example, "Time series"). Don't add the word "panel" or "visualization" to the title.
+- **Introduction:** Include an introduction that explains what the visualization type can help the user do.
+- **Main image:** Add a clear screenshot depicting the visualization displaying a typical use case.
```

Comment:
Gif maybe, but videos make me nervous for maintenance reasons. Images go out of date pretty quickly, and while it can be a big lift to update a bunch of screenshots if something in the UI changes, it's an enormous lift to update a bunch of videos. So I wouldn't recommend a video as the first visual aid that someone sees on the page.

Reply:
```suggestion
- **Main visual:** Add a clear screenshot depicting the visualization displaying a typical use case. A short video embed (5-10 seconds) is also acceptable. Don't use longer videos here.
```